server/liveblog/blogs/blogs.py

```diff
@@ -32,7 +32,7 @@
 
 blogs_schema = {
     'title': metadata_schema['headline'],
-    'description': metadata_schema['description'],
+    'description': metadata_schema['description_text'],
```

Comment:
It won't change the old blogs. In the `blogs` endpoint the name remains `description`